pages/knowledge/colours-and-contrast/text/README.md

```diff
@@ -0,0 +1,79 @@
+
+## Minimal contrast ratio
+
+The [Web Content Accessibility Guidelines (WCAG) specify a minimal contrast ratio](https://www.w3.org/TR/WCAG21/#contrast-minimum) of `4.5:1` against adjacent colour(s) for text in general. An exception is made for large-scale text: It is considered easier to read and can therefore be set in a lower contrast ratio of at least `3:1`.
+
+Large-scale text is defined as:
+
+- Either **bold** text of at least 14 points (or 18.5 pixels).
+- Or regular text of at least 18 points (or 24 pixels).
+
+(see [WCAG definition of “large scale (text)”](https://www.w3.org/TR/WCAG21/#dfn-large-scale))
+
+To illustrate this: the following image shows the full spectrum of possible contrast ratios from `1:1` on the left to `21:1` on the right.
+
+![Text with colour gradient overlay](_media/lazy-dog.png)
+
+Somewhere around the “x” the contrast ratio reaches `3:1`. Somewhere around the “m” we arrive at `4.5:1`. 
+
+For a more direct comparison: the following image shows both large and default text with contrast ratios of exactly `3:1` and `4.5:1`.
+
+![Small and large text](_media/small-and-large-text.png)
+
+It may not seem like much of a difference, but in fact it heavily enlargens the number of possible colour combinations.
+
+It is important to mention that the values `3:1` and `4.5:1` represent a *minimal* standard for *acceptable* contrast, they don’t represent an ideal. They should be treated as threshold and not as target; aiming higher is recommended. 
+
+## Enhancing text contrast
+
+Besides changing the colour of the text or the background there are many other ways do enhance contrast. For example:
+
+### Adding shadow
+
+Drop shadow around text can make up for insufficient contrast between text colour and background.
+
+For example, look at the following image. It shows two words, both with a red fill that has an insufficient contrast of `2.1:1` to the white background.
+
+![A word without and a word with shadow](_media/words-without-and-with-shadow.png)
+
+The shadow of the second word results in a contrast of appropriately `4.5:1`.
+
+### Adding a contrast layer
+
+In some cases, text is displayed on non-homogeneous backgrounds. Depending on the heterogeneity of the background, the text may be very hard to read.
+
+For example, look at the following image. It shows a white text “Welcome to the beach!” on blue sky with some white clouds. While there is decent contrast between white text and blue sky there is almost no contrast between white text and white clouds. Overall it’s very hard to read.
+
+![White text on blue sky and white clouds](_media/beach.png)
+
+You could increase contrast by adding a text shadow as described above. Or you could add a semi-transparent darkening layer between text and background
+
+![White text with semi-transparent darkening background](_media/beach-with-background.png)
+
+Of course you can also combine different solutions to add up their effects.
+
```

Comment:
Here I miss the sub-chapter "Adding graphical information" with info about e.g. highlighted text:

> For example, the following image shows a gray text, and some of the words are highlighted red to mark spelling mistakes...